terraform/shared/shared.tf.json.template.py

```diff
@@ -1014,6 +1014,29 @@ def conformance_pack(name: str) -> str:
                 ]
             }
         },
+        'aws_wafv2_regex_pattern_set': {
+            **{
+                name: {
+                    'name': config.qualified_resource_name(name),
+                    'scope': 'REGIONAL',
+                    **(
+                        {
+                            'lifecycle': {
+                                'ignore_changes': [
+                                    'regular_expression'
+                                ]
+                            }
+                        }
+                        if ignore_changes else
+                        {}
+                    )
+                }
+                for name, ignore_changes in [
+                    (config.blocked_user_agents_regex_term, True),
+                    (config.blocked_user_agents_custom_regex_term, False)
```

Comment:
I think this is the wrong way around.

```suggestion
                    (config.blocked_user_agents_regex_term, False),
                    (config.blocked_user_agents_custom_regex_term, True)
```